### ISSUE TITLE
VPAT -- Device Dependent Event Handler Alert

### ISSUE THREAD

**Issue by pbailie** [Submitty/Submitty#2566]
Per WAVE:
> **What It Means**
> An event handler is present that may not be accessible.
> **Why It Matters**
> The JavaScript events in use do not appear to be accessible to both mouse and keyboard users. To be fully accessible, critical JavaScript interaction should be device independent.
> **How to Fix It**
> Ensure that critical functionality and content is accessible by using a device independent event handler (which responds to both keyboard and mouse) or by using both a mouse dependent and a keyboard dependent event handler.
> **The Algorithm... in English**
> One of the following are present:
> - an `onmouseover` event but not an `onfocus` event
> - an `onclick` event on something other than a link, form control, or element with a `tabindex` value of 0
> - `ondblclick`
>
> **Standards and Guidelines**
> [Section 508 (l)](http://webaim.org/standards/508/checklist#standardl)

Views affected:
- Two subviews in the Homepage view
   - [x] Edit Name Form (two alerts)
   - [x] Change Password Form (two alerts)
- Three subviews in the Manage Students view
   - [x] Download Users (two alerts)
   - [x] Upload Classlist (two alerts)
   - [x] New User (two alerts)
- Three subviews in the Manage Graders View
   - [x] Download Graders (two alerts)
   - [x] Upload Grader List (two alerts)
   - [x] New Grader (two alerts)
- Edit Gradeable view
   - [ ] workflow buttons

(more views may be added as I continue VPAT analysis)

In the Homepage, Manage Students, and Manage Graders views, the alerts appear to be associated with popup handling code in `site/app/templates/generic/Popup.twig` -- one alert for line 5 and the second alert for line 7.
https://github.com/Submitty/Submitty/blob/428a100c460757059ec7bb0e106b02205e8291c1/site/app/templates/generic/Popup.twig#L5-L7
